packages/dx-vue-grid/docs/reference/data-type-provider.md

```diff
@@ -0,0 +1,70 @@
+# DxDataTypeProvider Plugin Reference
+
+A plugin that provides the capability to customize formatting options and editors depending on the data type.
+
+## Importing
+
+Use the following import statement:
+
+```js
+import { DxDataTypeProvider } from '@devexpress/dx-vue-grid';
+```
+
+## User Reference
+
+### Dependencies
+
+none
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+for | Array&lt;string&gt; | | The names of columns associated with the specified formatter and editor.
+formatterComponent? | [DxDataTypeProvider.ValueFormatter](#dxdatatypeprovidervalueformatter) | | A component that renders the formatted value.
```

Comment:
Wrong type/default

Reply:
I'm not sure about default value. It should by `undefined`. So, I used `DxDataTypeProvider.ValueFormatter` as a type.